doc/guide/packages.xml

```diff
@@ -310,6 +310,37 @@ mypackage/test.min.js.gz
       </varlistentry>
     </variablelist>
 
+    <para>The <code>command</code> and <code>environ</code> values can be dynamically
+    taken from a matching open command values. When a value in either <code>command</code>
+    or <code>environ</code> contains a variable wrapped in <code>${}</code>. The variable will be
```

Comment:
We also need to note how to escape and include a literal ```${XXX}``` in your command/environ.